backend/logs_helper/views.py

```diff
@@ -0,0 +1,93 @@
+import json
+import logging
+import os
+from datetime import datetime, timezone
+
+import redis
+from logs_helper.constants import LogsHelperKeys
+from logs_helper.exceptions import InvalidValueError, MissingFieldsKeyError
+from rest_framework import status, viewsets
+from rest_framework.decorators import action
+from rest_framework.exceptions import APIException
+from rest_framework.response import Response
+from utils.local_context import StateStore
+
+from .serializers import StoreLogMessagesSerializer
+
+logger = logging.getLogger(__name__)
+
+
+class LogsHelperView(viewsets.ModelViewSet):
+    """Viewset to handle all Tool Studio prompt related API logics."""
+
+    r = redis.Redis(
```

Comment:
We use [django-redis](https://pypi.org/project/django-redis/) already - let's try to reuse the same to remain error free.
Worst case, its not safe to create the redis connection as a class instance since the view might be shared by many threads.

Reply:
@chandrasekharan-zipstack Thanks for letting us know. The required changes have been made.